how-to/customize-workspace/public/manifest.fin.json

```diff
@@ -455,6 +466,54 @@
 				]
 			}
 		},
+		"dockProvider": {
+			"id": "customize-workspace",
+			"title": "Home Starter",
+			"icon": "http://localhost:8080/favicon.ico",
+			"workspaceComponents": {
+				"hideHomeButton": false,
+				"hideWorkspacesButton": false,
+				"hideNotificationsButton": false,
+				"hideStorefrontButton": false
+			},
+			"appTags": ["dock"],

```

Comment:
should this be apps: { 
"tooltip": "Apps",
"iconUrl": "http://localhost:8080/favicon.ico",
"tags":["dock"]
}

so they can customize the button that has the apps?

Reply:
I have changed the way this behaves now, you can have individual buttons or in a group